neo4j-to-neptune/docs/convert-csv.md

```diff
@@ -1,74 +1,141 @@
     NAME
             neo4j-to-neptune.sh convert-csv - Converts CSV file exported from Neo4j
-            via 'apoc.export.csv.all' to Neptune Gremlin import CSV files
-    
+            via 'apoc.export.csv.all' to Neptune Gremlin load data formatted CSV files,
+            and optionally automates the bulk loading of the converted data into Amazon Neptune.
+
     SYNOPSIS
-            neo4j-to-neptune.sh convert-csv {-d | --dir} <outputDirectory>
-                    {-i | --input} <inputFile> [ --infer-types ]
+            neo4j-to-neptune.sh convert-csv [ --bucket-name <bucketName> ]
+                    [ --bulk-load ] {-d | --dir} <outputDirectory>
+                    {-i | --input} <inputFile> [ --iam-role-arn <iamRoleArn> ]
+                    [ --infer-types ] [ --monitor ]
+                    [ --neptune-endpoint <neptuneEndpoint> ]
                     [ --node-property-policy <multiValuedNodePropertyPolicy> ]
+                    [ --parallelism <parallelism> ]
                     [ --relationship-property-policy <multiValuedRelationshipPropertyPolicy> ]
+                    [ --s3-prefix <s3Prefix> ]
                     [ --semi-colon-replacement <semiColonReplacement> ]
-    
+
     OPTIONS
+            --bucket-name <bucketName>
```

Comment:
Nit: It might be nice to combine `bucket-name` and `s3-prefix` into a single `s3-uri`/`s3-path` argument. Could be considered out of scope here, but it would be nice to have a followup task to simplify the arguments as much as possible.